stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -320,32 +375,38 @@ private void runInternal() throws Exception {
       mStartBarrierPassed = true;
 
       if (mParameters.mOperation == FuseIOOperation.LIST_FILE) {
+        for (String workerDirNames: mWorkerDirNames) {
+          for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
+            String dirPath = String.format("%s/%s/dir-%d", mParameters.mLocalPath,
+                workerDirNames, dirId);
+            File dir = new File(dirPath);
+            dir.listFiles();
+          }
+        }
+      } else if (mParameters.mOperation == FuseIOOperation.WRITE
+          || mParameters.mOperation == FuseIOOperation.LOCAL_READ) {
         for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
-          String dirPath = String.format("%s/%d", mParameters.mLocalPath, dirId);
-          File dir = new File(dirPath);
-          dir.listFiles();
+          for (int fileId = 0; fileId < mParameters.mNumFilesPerDir; fileId++) {
+            String filePath = String.format("%s/%s/dir-%d/file-%d", mParameters.mLocalPath,
+                mBaseParameters.mId, dirId, fileId);
+            processFile(filePath, isRead);
+          }
         }
-        return;
-      }
-
-      for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
-        for (int fileId = 0; fileId < mParameters.mNumFilesPerDir; fileId++) {
-          mCurrentOffset = 0;
-          String filePath = String.format("%s/%d/%d", mParameters.mLocalPath, dirId, fileId);
-          while (!Thread.currentThread().isInterrupted()) {
-            if (isRead && CommonUtils.getCurrentMs() > mContext.getEndMs()) {
-              closeInStream();
-              return;
-            }
-            long ioBytes = applyOperation(filePath);
-
-            // Done reading/writing one file
-            if (ioBytes <= 0) {
-              break;
-            }
-            // Start recording after the warmup
-            if (CommonUtils.getCurrentMs() > recordMs) {
-              mThreadCountResult.incrementIOBytes(ioBytes);
+      } else {
+        for (int dirRead = 0; dirRead < mWorkerDirNames.size(); dirRead++) {
```

Comment:
dirRead and dirId name are not informative enough
modify the name to show more clear info